go.mod

```diff
@@ -79,13 +79,13 @@ require (
 	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
```

Comment:
`go.mod` and `go.sum` don't need to be changed.

Reply:
done